src/main/java/common/DriverBinariesSetter.java

```diff
@@ -14,23 +14,31 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.security.GeneralSecurityException;
+import java.util.Base64;
 
 import static common.DriverBinariesSetter.Drivers.*;
 
 
 public final class DriverBinariesSetter {
 
-    private  static DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+    private static final DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+    private static final String sauceLabUserName =
+            new String(Base64.getDecoder().decode(ConfigurationManager.getProperty("sauceLab.username")));
+    private static final String sauceLabAccessKey =
+            new String(Base64.getDecoder().decode(ConfigurationManager.getProperty("sauceLab.access.key")));
 
```

Comment:
There is a duplication of the encoding mechanism. Please implement it in a separate helper Class with e.g. getEncodedValue()